sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,906 @@
+considerably slower than Normal mode, the design of Recovery mode ensures all
+sBTC can be redeemed so long as the Stacks blockchain and PoX are online.
+
+This proposal, if ratified, would be executed by at least two hard forks:  one
+to implement Normal mode, and a follow-on hard fork to implement Recovery mode.
+This SIP is designed to leverage features proposed by SIP-021, and can only
+activate if SIP-021 is ratified beforehand or concurrently with this SIP.
+
+This proposal describes Normal mode.  Recovery mode will be fully described in a
+future SIP, and will have a separate activation procedure.  The Recovery mode
+SIP cannot activate without this SIP first being ratified.
+
+# Specification
+
+The proposed SIP-021 upgrade would do away with Stacks forks, which vastly
+simplifies the sBTC design.  Importantly, miners and Stackers cannot "roll back"
+sBTC operations, meaning that peg-in and peg-out operations follow at-most-once
+semantics.  This lack of "fork risk" means that the system can support
+arbitrarily large amounts of BTC pegged-in and pegged-out, assuming that at
+least 70% of Stackers by weight are trustworthy.
+
+The sBTC system state is materialized in a boot contract called `.sbtc`.  This
+contract, like `.pox-4`, is managed in part by the node itself.  In particular,
+the node writes sBTC state from Bitcoin into the data space of `.sbtc` (such as
+via a private function that only the node can call), so that it can be accessed
+by smart contracts in Stacks.
+
+The following sections describe the Normal mode of operation.  Recovery mode is
+left for future work, but is described in a separate section after Normal mode.
+
+## Peg-In
+
+A peg-in operation is initiated as a Bitcoin transaction with a special payload
+in an `OP_RETURN` output (Figure 1).  Because this is a Stacks-on-Bitcoin
+transaction, a miner in a subsequent tenure is required by the consensus rules
+to process it as part of producing the first block in its tenure.  The Stacks
+node records the existence of this peg-in operation in the data space of the
+`.sbtc` smart contract, where it is marked as a _pending_ operation that
+Stackers must act upon.  Namely, the relevant parts of the transaction are
+extracted and translated in to Clarity values and made available via internal data maps.
+
+Stackers, via their signer daemons, already subscribe to the Stacks blockchain
+for events.  The act of writing new sBTC peg-in state to the `.sbtc` contract
+generates such an event -- an event that alerts the signer daemons to begin a
+signing round to act on the peg-in.
+
+In the happy path, at least 70% of Stackers agree that the peg-in transaction is
+valid, and they generate a contract-call to the `.sbtc` contract to
+_authorize_ the peg-in by minting the equivalent number of sBTC tokens to
+an account address of the user's choice.  Crucially, the transaction is signed 
+collectively by Stackers via a WSTS signing round -- it is authenticated via a
```

Comment:
[nit] first mention of WSTS, can you include a link or pointer for further reading?